gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* It is difficult for managers to approximate what the time/effort commitment will be for employees taking on InnerSource project tasks.
+* The less insight a manager has into the effort and accomplishment of their employees in InnerSource projects, the more unlikely it is that he can reward their employees for their InnerSource work.
+* The transparency and reputational aspect of the code submissions within a large company should work to keep the code quality high for InnerSource contributions.
+* Individual employees may commit themselves to InnerSource project tasks without first getting approval from their manager or having the skills or time required to complete it.
+
+## Context
+
+* Part of a manager's job is tracking, estimating and understanding use of employee time
+* Managers have given tacit acceptance that their employees should take part in InnerSource projects
+* There is no corporate policy on employee’s involvement in InnerSource projects
+* Employees desire the freedom to work on projects that are of interest to or would professionally benefit them
+* Employees desire being recognized for their efforts when contributing to InnerSource projects
+* Employees desire exposure to different areas within and to work with a larger pool of people across the company
+
+## Solutions
+
+Create a “gig” based intranet site where individuals can advertise their skills and areas of interest and project owners can advertise opportunities for collaboration.  Each gig should describe exactly the skill and time requirements needed so those can be easily and clearly communicated to an employee’s direct management.  A points based system should be created to reward and track an employee’s involvement in a gig. Points accrued by completing a gig can then be used as performance management criteria and to garner insight in to the areas of expertise that exist within an organization. All gigs are initially vetted with the Gig owner to determine that the employee has the prerequisite skills and allotted time and needed access to complete the gig.  Gigs can also factor in soft skills such as garnering help organizing a group event or requesting help editing/writing a report or requests for mentorship etc.
```

Comment:
Are there some examples you could provide for how you are mapping points to work? Also, is the Gig owner responsible for awarding the points upon successful completion of the gig? Or is there discretion for rewarding points along the way to completion?

Reply:
We could include the specifics of how the points are being awarded in our implementation.  Again, we thought that this level of detail was too implementation specific and better left to those wishing to adopt the pattern.  Again, this is all up for debate.

Reply:
I think this particular detail would be helpful to those looking to adopt this pattern.

Reply:
Understood.  Will update this language accordingly.